code/modules/bitrunning/virtual_domain/virtual_domain.dm

```diff
@@ -74,12 +74,33 @@
 	var/list/custom_spawns = list()
 	/// Set TRUE if you want reusable custom spawners
 	var/keep_custom_spawns = FALSE
-
+	/// At least this many candidates or it will cancel the loading!
+	var/mission_min_candidates = 0
+	/// Maximum amount possible of above.
+	var/mission_max_candidates = 0
+	/// Ghosts that will be spawned as, presumably, an antagonist in the map.
+	var/list/chosen_ghosts
+	/// List of spawners used for candidates.
+	var/list/obj/effect/mob_spawn/ghost_role/ghost_spawners
+	var/list/mob/living/ghost_mobs
```

Comment:
Undocc'd. 

Also if there are not meant to be set by subtypes I would recommend `VAR_PRIVATE` / `VAR_FINAL`.

Reply:
It showed a warning so i removed the privating